### DISCUSSION TITLE
Live Preview function in Multilingual Craft CMS website

### DISCUSSION THREAD

**Discussion by dolphiq-allard** [craftcms/cms#10107] [Q&A] [answered]
Struggling a bit with a multilingual Craft CMS website. I am trying to get Live Preview working on both languages I use on my site. The Primary language works just fine, live preview is showing, and all edits are visible. However, if I switch to the secondary language and try to preview I get thre primary version.

In my .env file I setup two environmental variables the following URL's which I use in "Settings > Sites > Base URL":

`PRIMARY_SITE_URL=http://mysite.local:8080`
`PRIMARY_SITE_URL_EN=http://mysite.local:8080/en/`

In my Sections settings, I configured it as shown below:
![sections](https://user-images.githubusercontent.com/4312073/142183672-8cf8c561-6bab-4cf5-9ba1-e92a02a01e47.png)

I tried changing the second row in "Preview Targets" to `en/news/{slug}` however this was without success.

Any help would be much appreciated. Thanks much.

**Comment by brandonkelly** [comment, marked as answer]
Just a single preview target should be necessary, with its URL Format set to `{url}`. It will show the entry in whichever site is currently being edited.

**Comment by dolphiq-allard** [comment]
Thanks @brandonkelly this worked like a charm. I was clearly thinking way too complex.

**Reply by brandonkelly** [reply]
Hah no worries, glad that worked!
